Source/Scene/FrameState.js

```diff
@@ -288,10 +288,6 @@ define([
         this.minimumDisableDepthTestDistance = undefined;
     }
 
-    FrameState.prototype.addCommand = function(command) {
```

Comment:
@bagnell heads up that you might need to make a small tweak when this is merged into the vector tile branch.